Add code for implementing tracklet based object detection.

SriramPingali opened the pull request:
Attempt to solve the issue of jittering of images. And integrates tracking and detection adhering to time-based data collection.

SriramPingali committed:
Merge pull request #1 from dsciitpatna/master

Update README.md

SriramPingali committed:
Add Code for implementing tracklet based object detection.

piyushchauhan commented:
@SriramPingali Make a before and after videos so that anyone can differentiate that tracking is being done. This will also help us in documnetation purposes also.

SriramPingali committed:
Add code for YOLO V3 object detection.

Yolo V3 object detection algorithm which creates a CSV database parallelly on its own.

SriramPingali committed:
Deleted Traclet based object detection code file, will upload a comprehensive folder later on.

SriramPingali committed:
Update READ.me
